hw/drivers/adc/gpadc_da1469x/src/gpadc_da1469x.c

```diff
@@ -30,6 +30,10 @@
 
 static struct da1469x_gpadc_dev *da1469x_gpadc_dev;
 
+// 1/0.180265 * 1024 = 5680, 1/0.06907 * 1024 = 14825, 1/0.065067 * 1024 = 15737
+// 1/0.068603 * 1024 = 14926, 1/0.066215 * 1024 = 15464, 1/0.170288 * 1024 = 6013
```

Comment:
Can you please use C style comments instead of C++ style comments as per our coding style doc.